### PR TITLE
Created basic web structure, added all required dependencies

### PR DESCRIPTION
**Pull request by Faradey27**

### CODE REVIEW

**Comment by Faradey27** [comment]
@rasendubi

File: LICENSE [hunk 1]
```diff
@@ -1,35 +1,35 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
+GNU AFFERO GENERAL PUBLIC LICENSE
```
Comment (rasendubi) [inline LICENSE]: Please, revert license changes

File: README.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+* [Node 7.1.0 or later](https://nodejs.org/en/)
+* [Npm 3.10.9 or later](https://docs.npmjs.com/)
+* Unix based operation system([os x](https://en.wikipedia.org/wiki/OS_X), [ubuntu](https://en.wikipedia.org/wiki/Ubuntu_(operating_system)), [fedora](https://en.wikipedia.org/wiki/Fedora_(operating_system)), etc.)
+
+Run the following commands to verify that you have the correct versions of Node and NPM installed:
+
+    node -v
+    npm -v
+### (b) Clone the Repository
+    git clone git@github.com:Faradey27/Heutagogy.git Heutagogy
```
Comment (rasendubi) [inline README.md]: Hard-coded your own repo

File: docs/basic/README.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+* [Node 7.1.0 or later](https://nodejs.org/en/)
+* [Npm 3.10.9 or later](https://docs.npmjs.com/)
+* Unix based operation system([os x](https://en.wikipedia.org/wiki/OS_X), [ubuntu](https://en.wikipedia.org/wiki/Ubuntu_(operating_system)), [fedora](https://en.wikipedia.org/wiki/Fedora_(operating_system)), etc.)
+
+Run the following commands to verify that you have the correct versions of Node and NPM installed:
+
+    node -v
+    npm -v
+### (b) Clone the Repository
+    git clone git@github.com:Faradey27/Heutagogy.git Heutagogy
```
Comment (rasendubi) [inline docs/basic/README.md]: same here

**Review by rasendubi — COMMENTED** [review]
I'm a bit afraid we don't need all this stuff. My initial idea was to use `heutagogy-website` repo for the presentation-like website. That means it's most likely is a collection of static web pages. So I'm a bit reluctant to merge this.

Do you have another vision?

**Comment by Faradey27** [comment]
@rasendubi i thought that in future we will have more complex application with permissions system, maybe some statistic(charts, tables etc.), other different clever things. If you sure that we will have just static display for saved article it's ok, in such case i will left only 1-2 js files and react lib.

about html files, you want server side rendering? or what the reason?

**Comment by rasendubi** [comment]
I mean the site is the front page of the project. (Similar to this page: https://atom.io/.) It's just a static page with no backend logic.

I think you're talking about the frontend application (the one which interacts with the backend). Should we separate them?

**Comment by Faradey27** [comment]
i m understood, currently you are looking on frontend application,

yea, i think it's good idea to have separate repo for landing and for application
@rasendubi

**Comment by rasendubi** [comment]
Created https://github.com/heutagogy/heutagogy-frontend

**Comment by rasendubi** [comment]
If I properly understand, this also brings in a node.js backend. Is that ok given that we already have one?

**Comment by Faradey27** [comment]
@rasendubi nodejs required only for development, it's give nice features like hot reload(https://facebook.github.io/react-native/blog/2016/03/24/introducing-hot-reloading.html), more clever debug, etc. and this node server just serve frontend files

for production we can use anything, like our python server)

**Comment by rasendubi** [comment]
I see. Thanks for explanation!

**Pull request closed without merging**